packages/dashboard-v2/src/components/Slider/useActiveBreakpoint.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useMemo, useCallback, useState } from "react";
+import { useWindowSize } from "react-use";
+
+export default function useActiveBreakpoint(breakpoints) {
+  const { width: windowWidth } = useWindowSize();
+  // Since our breakpoints are setup with min-width rule, we need to sort them from largest to smallest
+  const monitoredBreakpoints = useMemo(
+    () => breakpoints.slice().sort(({ minWidth: widthA }, { minWidth: widthB }) => widthB - widthA),
+    [breakpoints]
+  );
+  const findActiveBreakpoint = useCallback(
+    () => monitoredBreakpoints.find((breakpoint) => windowWidth >= breakpoint.minWidth),
+    [monitoredBreakpoints, windowWidth]
+  );
+
+  const [activeBreakpoint, setActiveBreakpoint] = useState(findActiveBreakpoint());
+
+  useEffect(() => {
+    setActiveBreakpoint(findActiveBreakpoint());
+  }, [findActiveBreakpoint]);
```

Comment:
in the follow up, separate pull request, let's include https://www.npmjs.com/package/eslint-plugin-react-hooks and scan the whole project with it

Reply:
It's already a part of the project, as `.eslintrc` extends `react-app` config. Adding `setActiveBreakpoint` is not necessary here, as React guarantees it to be stable ([reference](https://reactjs.org/docs/hooks-reference.html#usestate)):
<img width="952" alt="image" src="https://user-images.githubusercontent.com/3853033/155763877-50898493-a819-4316-832c-76209a08881b.png">

Reply:
perfect